apps/meta-data-utils/test/tableQuery.test.ts

```diff
@@ -1,11 +1,8 @@
 import { describe, it, expect } from "vitest";
+import { catalogueMetadata, catalogueOntologies, petStoreMetadata } from "./test-resources/metadata";
```

Comment:
autolayout wants to put this on multiple lines, so I would expect that to change back at a later pr

Reply:
a check i think the /test folder is not included in the format job , ill try to fix it

Reply:
add the test folder to the format script path